src/Graphics/src/Graphics/Graphics.csproj

```diff
@@ -4,7 +4,6 @@
     <TargetFrameworks>netstandard2.1;netstandard2.0;$(_MauiDotNetTfm);$(MauiGraphicsPlatforms);$(_MauiPreviousDotNetTfm);$(MauiGraphicsPreviousPlatforms)</TargetFrameworks>
     <!-- Optional: Build symbol package (.snupkg) to distribute the PDB containing Source Link -->
     <IncludeSymbols>true</IncludeSymbols>
-    <SymbolPackageFormat>snupkg</SymbolPackageFormat>
```

Comment:
@mattleibow  It s is generating them , but I needed to remove it from here. 

But the are on the BlobArtifacts folder https://dev.azure.com/dnceng/7ea9116e-9fac-403d-b258-b31fcf1bb293/_apis/build/builds/2535536/artifacts?artifactName=BlobArtifacts&api-version=7.1&%24format=zip